### PR TITLE
add more legacy addresses for eth, etc and xrp

### PR DESCRIPTION
**Pull request by amougel**
I added the missing legacy derivations from chrome apps
These are single addresses (no iteration possible)

### CODE REVIEW

File: src/derivation.js [hunk 1]
```diff
@@ -36,9 +36,21 @@ export const getDerivationScheme = ({
     // old MEW derivation scheme for ETC
     return "44'/60'/160720'/0'/<account>";
   }
-  if (derivationMode === "rip") {
```
Comment (gre) [inline src/derivation.js]: This must not disappear, otherwise will create regression in existing accounts.

File: src/derivation.js [hunk 2]
```diff
@@ -36,9 +36,21 @@ export const getDerivationScheme = ({
...
-    // XRP legacy that the old Chrome Ripple Wallet used to wrongly derivate addresses on.
-    return "44'/144'/0'/<account>'";
+  if (derivationMode === "ripWalletIdentifier") {
```
Comment (gre) [inline src/derivation.js]: Basically call this one rip for backward compatible

File: src/derivation.js [hunk 3]
```diff
@@ -36,9 +36,21 @@ export const getDerivationScheme = ({
...
+    // XRP legacy that the old Chrome Ripple Wallet used to wrongly derivate addresse on.
+    return "44'/144'/0'/0'";
+  }
+  if (derivationMode === "ripWalletMetaPassword") {
```
Comment (gre) [inline src/derivation.js]:
```suggestion
  if (derivationMode === "rip2") {
```

**Comment by amougel** [comment]
1. > I think Ledger Live will expect all derivations to be iterable

That's what I thought so I looked into the code and realized that there just a `replace` occuring on 
the path string (for account). So it should not break

Regarding the comment they reflect the reality of the chrome app

**Comment by gre** [comment]
i'll merge this and bring an iteration function in another PR

**Pull request merged**